scripts/linters/python_linter.py

```diff
@@ -320,6 +341,35 @@ def all_filepaths(self):
         """Return all filepaths."""
         return self.files_to_lint
 
+    @staticmethod
+    def _get_trimmed_error_messages(lint_messages):
+        """Remove extra bits from pylint error messages.
+
+        Args:
+            lint_messages: list(str). Messages returned by the python linter.
+
+        Returns:
+            str. A string with the trimmed error messages.
+        """
+        error_messages = ''
+        # Remove newlines and coverage report from the end of message.
+        # we need to remove last five items from the list becuase starting from
+        # end we have three lines with just newline characters, fourth line
+        # contains the coverage report i.e.
+        # Your code has been rated at 9.98/10 (previous run: 10.00/10, -0.02)
+        # and one line with dashes(---).
+        lint_messages = lint_messages[:-5]
+        for message in lint_messages:
+            # Every pylint message has a message id inside the brackets
+            # so we need to check if it is true and if case is true then
```

Comment:
When you say "if it is true", what does "it" refer to?

If you mean "whether the pylint message has a message id", doesn't that contradict your earlier statement that "every pylint message has a message id"?